### ISSUE TITLE
GroupSAMLLinks.remove not working as it should

### ISSUE THREAD

**Issue by czi** [jdalrymple/gitbeaker#3467]
**Description**

- Node.js version: 18.18.2
- Gitbeaker version:
- Gitbeaker release (cli, node, browser, core, requester-utils): 39.24.0 
- OS & version: Darwin

I think there may be a bug in the remove code.  Create is working as expected, but delete is giving "Not Found" errors.

code snippet:
```
     console.log(group.id)
      console.log(group.saml_group_links)
      let r = await tGlab.GroupSAMLLinks.remove(group.id,'GitLabTestGroup')
```

console snippet:
```
78519941
[
  { name: 'GitLabTestGroup', access_level: 20 },
]
Not Found
```

(Paste any relevant logs - please use code blocks (```) to format console output,
logs, and code as it's very hard to read otherwise.) -->

**Steps to reproduce**
Create any SAML Group and try to delete it using the library.

**Expected behaviour**
SAML Group is found and deleted.

**Actual behaviour**

Not finding the group and not deleting it.

**Possible fixes**

**Checklist**
- [ X ] I have checked that this is not a duplicate issue.
- [ X ] I have read the documentation.

**Comment by czi** [comment]
There may be an issue with GroupSAMLLinks.show too which may be part of the problem.  I'm fairly sure this should just be returning the information about the single SAML group.

```
      tGlab.GroupSAMLLinks.show(group.id, 'GitLabTestGroup', {})

```
```
[
  { name: 'GitLabTestGroup', access_level: 20 },
  { name: 'Test2', access_level: 10 }
]

```

**Comment by jdalrymple** [comment]
Yea i made another typo, fixing now!

**Comment by jdalrymple** [comment]
:rocket: Issue was released in [`39.25.0`](https://github.com/jdalrymple/gitbeaker/releases/tag/39.25.0) :rocket:
